packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -636,6 +679,18 @@ class _RailDestination extends StatelessWidget {
         );
         break;
       case NavigationRailLabelType.all:
+        final Widget iconWithIndicator =  Stack(
+          alignment: Alignment.center,
+          children: <Widget>[
+            if (useIndicator)
+              NavigationIndicator(
+                animation: destinationAnimation,
+                width: 56,
+                borderRadius: 100,
+              ),
```

Comment:
Same nit as above, the 100 border radius feels weird to me

Reply:
Corrected value.